operations/helm/charts/mimir-distributed/README.md

```diff
@@ -66,6 +66,8 @@ As part of this chart many different pods and services are installed which all
 have varying resource requirements. Please make sure that you have sufficient
 resources (CPU/memory) available in your cluster before installing Grafana Mimir Helm Chart.
 
+For a more detailed guide on setting up Grafana Mimir refer to [Getting started with Grafana Mimir using the Helm chart](https://grafana.com/docs/mimir/latest/operators-guide/deploying-grafana-mimir/getting-started-helm-charts/).
```

Comment:
```suggestion
For details about setting up Grafana Mimir, refer to [Get started with Grafana Mimir using the Helm chart](https://grafana.com/docs/mimir/latest/operators-guide/deploy-grafana-mimir/get-started-helm-charts/).
```
Some of these changes are likely out of scope for this particular PR. Please take at least the first part of the suggestion.